entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md

```diff
@@ -1133,6 +1133,28 @@ This change simplifies the implementation of the aforementioned interfaces.
 
 Use the new extension methods.
 
+## Provider-specific Metadata API changes
+
+[Tracking Issue #214](https://github.com/aspnet/EntityFrameworkCore/issues/214)
+
+This change will be introduced in EF Core 3.0-preview 6.
```

Comment:
Nit: I suggest we go with with present tense, which seems to me more temporally neutral, so that we don't need to change it anymore, e.g.:

This change is introduced in EF Core 3.0-preview 6.

Incidentally, I think there is a guideline for docs that recommends just using present tense.